templates/README_guide.md

```diff
@@ -0,0 +1,170 @@
+
+Describe any criteria for splitting the data, if used. If their are differences between the splits (e.g. if the training annotations are machine-generated and the dev and test ones are created by humans, or if different numbers of annotators contributed to each example), describe them here.
+
+Provide the sizes of each split. As appropriate, provide any descriptive statistics for the features, such as average length.  For example:
+
+|                            | Tain   | Valid | Test |
+| -----                      | ------ | ----- | ---- |
+| Input Sentences            |        |       |      |
+| Average Sentence Length    |        |       |      |
+
+## Dataset Creation
+
+### Curation Rationale
+
+What need motivated the creation of this dataset? What are some of the reasons underlying the major choices involved in putting it together?
+
+### Source Data
+
+This section describes the source data (e.g. news text and headlines, social media posts, translated sentences,...)
+
+#### Initial Data Collection and Normalization
+
+Describe the data collection process. Describe any criteria for data selection or filtering. List any key words or search terms used. If possible, include runtime information for the collection process.
+
+If data was collected from other pre-existing datasets, link to source here and to their [Hugging Face version](https://huggingface.co/datasets/dataset_name).
+
+If the data was modified or normalized after being collected (e.g. if the data is word-tokenized), describe the process and the tools used.
+
+#### Who are the source language producers?
+
+State whether the data was produced by humans or machine generated. Describe the people or systems who originally created the data.
+
+If available, include self-reported demographic or identity information for the source data creators, but avoid inferring this information. Instead state that this information is unknown. See [Larson 2017](https://www.aclweb.org/anthology/W17-1601.pdf) for using identity categories as a variables, particularly gender.
+
+Describe the conditions under which the data was created (for example, if the producers were crowdworkers, state what platform was used, or if the data was found, what website the data was found on). If compensation was provided, include that information here.
+
+Describe other people represented or mentioned in the data. Where possible, link to references for the information.
+
+### Annotations
+
+If the dataset contains annotations which are not part of the initial data collection, describe them in the following paragraphs.
+
+#### Annotation process
+
+If applicable, describe the annotation process and any tools used, or state otherwise. Describe the amount of data annotated, if not all. Describe or reference annotation guidelines provided to the annotators. If available, provide interannotator statistics. Describe any annotation validation processes.
```

Comment:
What should people do if they can't fill in some paragraph? Add `N/A`?

Reply:
They should put `N/A` if there is no information in the paragraph or leave `[Needs More Information]` if they simply don't know, in that case we'll eventually want to reach out to the authors or explain why the information is missing.